### ISSUE TITLE
Axe updates rejecting the accordion section 

### ISSUE THREAD

**Issue by colinbruce** [x-govuk/govuk-components#452]
https://github.com/x-govuk/govuk-components/blob/d888b9cc3f41f68a9024978a30b8d171ffeaed96/app/components/govuk_component/accordion_component/section_component.html.erb#L10

This line adds an aria-labelledby attribute, but this mornings AXE gem updates now raise the following error:
```
aria-labelledby attribute cannot be used on a div with no valid role attribute
```

changing it to 
```ruby
<%= tag.div(content, id: id(suffix: 'content'), role: 'region', class: "#{brand}-accordion__section-content", aria: { labelledby: id }) %>
```

passes the axe tests, but I am not an expert on Aria roles and definitions!

**Comment by colinbruce** [comment]
See #453 for a potential fix, but happy to be guided by accessibility experts :+1:

**Comment by frankieroberto** [comment]
@colinbruce thanks for raising this. Would you mind re-posting this as [an issue on `govuk-frontend`](https://github.com/alphagov/govuk-frontend/issues) (and linking to this one)?

The HTML for the accordion in this ruby code is based on the HTML in govuk-frontend, so the Design System team will be best placed to evaluate whether this is a genuine accessibility issue or a false-positive.

**Comment by peteryates** [comment]
Closing this as [it should be fixed](https://github.com/alphagov/govuk-design-system/issues/3508) upstream.
